packages/lib/server/repository/user.ts

```diff
@@ -624,6 +625,7 @@ export class UserRepository {
               },
             }
           : undefined,
+        creationSource: CreationSource.WEBAPP,
```

Comment:
why are you hardcoding this in user repository?

Reply:
Some other function might call this repository so we are not always sure that the creation source is WEBAPP

Reply:
Yeah fair point! Let me pull that from the source